<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1098</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>600</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: rgb(61, 61, 61);
border-radius: 20px;
}

QLabel{
font: 900 9pt &quot;System&quot;;
color: rgb(0, 255, 255);
border-radius: 5px;
background-color: rgb(64, 65, 66);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>800</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt;</string>
       </property>
       <property name="text">
        <string>Encriptador de Textos</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>490</width>
         <height>500</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>490</width>
         <height>500</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>490</width>
         <height>500</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>450</width>
          <height>27</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 15pt;
color: white;</string>
        </property>
        <property name="text">
         <string>Ingrese el texto a Encriptar...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textEncriptar">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>451</width>
          <height>351</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(215, 255, 238);
font: 13pt &quot;SansSerif&quot;;
padding: 20px;</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>420</y>
          <width>440</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>440</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>440</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0.238545, y1:0.562, x2:1, y2:0.625, stop:0.0391061 rgba(129, 54, 83, 255), stop:0.903409 rgba(104, 53, 0, 255));
color: white;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0.176045, y1:0.556, x2:1, y2:0.625, stop:0.0391061 rgba(147, 129, 160, 255), stop:0.903409 rgba(104, 53, 0, 255));
transform: scale(1,1);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="encriptar">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Encriptar</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>block.png</normaloff>block.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="desencriptar">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Desencriptar</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>desblock.png</normaloff>desblock.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>80</y>
         <width>350</width>
         <height>500</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>500</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>500</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTextBrowser" name="textDesencriptar">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>300</width>
          <height>401</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(215, 255, 240);
font: 13pt &quot;SansSerif&quot;;
text-align: center;
padding: 20px;</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QFrame" name="frame_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>430</y>
          <width>320</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0.238545, y1:0.562, x2:1, y2:0.625, stop:0.0391061 rgba(129, 54, 83, 255), stop:0.903409 rgba(104, 53, 0, 255));
color: white;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0.176045, y1:0.556, x2:1, y2:0.625, stop:0.0391061 rgba(147, 129, 160, 255), stop:0.903409 rgba(104, 53, 0, 255));
transform: scale(1,1);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="copiaText">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0.238545, y1:0.562, x2:1, y2:0.625, stop:0.0391061 rgba(129, 54, 83, 255), stop:0.903409 rgba(104, 53, 0, 255));
color: white;</string>
           </property>
           <property name="text">
            <string>Copiar Texto</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>copy.png</normaloff>copy.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QPushButton" name="btnClose">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>30</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0.238545, y1:0.562, x2:1, y2:0.625, stop:0.0391061 rgba(129, 54, 83, 255), stop:0.903409 rgba(104, 53, 0, 255));
color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>close.png</normaloff>close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
